lib/board.rb

```diff
@@ -0,0 +1,81 @@
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
why are we setting @cells there and then call reset! that does that again for us?